.github/workflows/wazuh-build-push-docker-action.yml

```diff
@@ -0,0 +1,119 @@
+# This is a basic workflow that is manually triggered
+
+name: Manual workflow build and push docker image
+
+# Controls when the action will run. Workflow runs when manually triggered using the UI
+# or API.
+on:
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      build-manager-image:
+        type: boolean
+        description: 'Build manager image'
+        required: false
+      wazuh-manager-version: 
+        description: 'Wazuh manager version'
+        default: 'v4.3.4-rc1'
```

Comment:
Why this one uses an RC tag? This tag might no longer exist (in fact, it doesn't).

Also, I'd update the default version to the latest published one, which is 4.3.8, for every action.

Reply:
Done